gradle.properties

```diff
@@ -1,5 +1,9 @@
-version=4.8.0-SNAPSHOT
```

Comment:
have the dependencies being check for vulnerabilities?

Reply:
I did not change any dependency version just moved the version of the project itself to the build.gradle.kts file